### DISCUSSION TITLE
[Q&A]  Native inventory vs plugin

### DISCUSSION THREAD

**Discussion by waaalex** [glpi-project/glpi-agent#615] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello,
What is best practice? Use glpi native inventory or using glpi plugin inventory?
Thanks.

**Comment by g-bougard** [comment, marked as answer]
Hi @waaalex 

your question demonstrates people can still be confused, but indeed you're also using native inventory when you're using glpi-inventory plugin.

Until GLPI 9, automatic inventory support was provided by FusionInventory plugin, that part was migrated to GLPI as native inventory. But as the FI plugin was providing some other features, we provided glpi-inventory to support that features missing in GLPI. That features are only to support other tasks than inventory: netdiscovery, netinventory, collect, deploy & ESX.

If you need to use at least one of these tasks, you'll have actually to setup `server` configuration in GLPI-Agent with an URL pointing to glpi-inventory plugin. But I can confirm you, the plugin will indeed internally pass the inventory to native inventory when it receives an inventory from an agent.

I can understand the confusing factor is the required server url to be configured on agent-side. Maybe this will change in the future.

**Reply by waaalex** [reply]
Thank you @g-bougard :)
